packages/twenty-server/src/engine/core-modules/labs/utils/is-public-feature-flag.util.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { FeatureFlagKey } from 'src/engine/core-modules/feature-flag/enums/feature-flag-key.enum';
+import { isPublicFeatureFlag } from 'src/engine/core-modules/labs/utils/is-public-feature-flag.util';
+
+describe('isPublicFeatureFlag', () => {
+  it('should return true for public flags', () => {
+    expect(isPublicFeatureFlag(FeatureFlagKey.IsCommandMenuV2Enabled)).toBe(
```

Comment:
Probably not a good idea to write a test that depends on a feature flag that will be removed